sys/net/network_layer/ng_ipv6/ng_ipv6.c

```diff
@@ -66,6 +67,11 @@ kernel_pid_t ng_ipv6_init(void)
     return _pid;
 }
 
+kernel_pid_t ng_ipv6_getpid(void)
+{
+    return _pid;
+}
```

Comment:
To quote @Kijewski once again: this is not Java. Just make `_pid` a global variable.